src/main/java/fr/jmmc/oitools/model/OIFitsFile.java

```diff
@@ -803,8 +801,8 @@ public boolean isOIFits2() {
     }
 
     /**
-     * define the version of the file
-     * @param content information for find the version of the file
+     * Return the OIFitsStandard for an OIFitsFile
```

Comment:
Fix 'Return the OIFitsStandard for an OIFitsFile' by
'Get the OIFitsStandard corresponding to the given keyword content value'